CHANGES.md

```diff
@@ -51,6 +51,13 @@ change is described below.
 
 [#240]: https://github.com/nextstrain/cli/pull/240
 
+* Local images used in a narrative are now automatically embedded into it when
+  uploading the narrative to nextstrain.org via `nextstrain remote upload`.
+  This permits local authoring of narratives that reference images on the local
```

Comment:
I'd clarify here that "local authoring" means in a markdown renderer and that relative images will not work in auspice.